app/views/easy_tokens/tokens/show.html.erb

```diff
@@ -1,24 +1,31 @@
-<p id="notice"><%= notice %></p>
```

Comment:
I think we can remove `show` view completely. Let's modify the controller, so user is redirected to `index` after token is created. makes sense?